### PR TITLE
Added: Import_Clementine_to_Strawberry.sh

### PR DESCRIPTION
**Pull request by vazhnov**
Based on SQL commands from
https://github.com/strawberrymusicplayer/strawberry/wiki/Import-collection-library-and-playlists-data-from-Clementine

Script works in Gentoo.

* I'm not sure about correct path to put this script in.
* Please check English grammar.

### CODE REVIEW

**Comment by vazhnov** [comment]
These are no need in this script, I can remove them:
```shell
set -o pipefail
shopt -s dotglob
```

**Comment by vazhnov** [comment]
TODO: update wiki page when this will be merged.

**Comment by jonaski** [comment]
Thanks. Look good to me, but we should simplify the name of the script. I suggest: `import-from-clementine.sh`

File: dist/scripts/Import_Clementine_to_Strawberry.sh [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+# Use hardcoded path if no parameters. No need in quotes here! See `man bash`, "Parameter Expansion":
+FILE_SRC=${1:-~/.config/Clementine/clementine.db}
+FILE_DST=${2:-~/.local/share/strawberry/strawberry/strawberry.db}
+
+test -f "$FILE_SRC" || { echo "No file: $FILE_SRC"; exit 1; }
+test -f "$FILE_DST" || { echo "No file: $FILE_DST"; exit 1; }
+
+echo "Will try to copy information from $FILE_SRC to $FILE_DST."
+echo
+echo 'This scipt will **delete all information** from Strawberry database!'
```
Comment (jonaski) [inline dist/scripts/Import_Clementine_to_Strawberry.sh:38]: script

File: dist/scripts/Import_Clementine_to_Strawberry.sh [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+set -o nounset
+set -o errexit
+set -o pipefail
+shopt -s dotglob
+
+# Use hardcoded path if no parameters. No need in quotes here! See `man bash`, "Parameter Expansion":
+FILE_SRC=${1:-~/.config/Clementine/clementine.db}
+FILE_DST=${2:-~/.local/share/strawberry/strawberry/strawberry.db}
+
+test -f "$FILE_SRC" || { echo "No file: $FILE_SRC"; exit 1; }
```
Comment (jonaski) [inline dist/scripts/Import_Clementine_to_Strawberry.sh:33]: No such file:

**Review by jonaski — CHANGES_REQUESTED** [review]

**Review by jonaski — APPROVED** [review]

**Pull request closed without merging**